README.md

```diff
@@ -189,6 +189,18 @@ The EncryptedCardData is encrypted according to the OAEP padding scheme. Once de
   }
 }
 
+### Failed callback
+
+In case that the user rejects the payment and the payment times out, a failed callback is sent to the FailedPaymentCallbackUrl url supplied in the initiate payment call.
+
+```
+{
```

Comment:
Can't remember how the reset is documented, but you might want to indicate this is a response body with a status code like sample below. This is also fine. 

```
Status: 400
Body: 
{
    'code': '100'
    ...
}
```

Reply:
It is a request body to the above url